{
  "Header": {
    "Original": "CVE-2025-2998 (A vulnerability was found in PyTorch 2.6.0. It has been declared as cr ...)",
    "ID": "CVE-2025-2998",
    "Description": "(A vulnerability was found in PyTorch 2.6.0. It has been declared as cr ...)"
  },
  "Annotations": [
    {
      "Original": "- pytorch \u003cunfixed\u003e",
      "Type": "package",
      "Package": "pytorch",
      "Kind": "unfixed"
    },
    {
      "Original": "[bookworm] - pytorch \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "pytorch",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/pytorch/pytorch/issues/149622",
      "Type": "NOTE",
      "Description": "https://github.com/pytorch/pytorch/issues/149622"
    }
  ]
}